manual/numfocus-coc.md

```diff
@@ -0,0 +1,238 @@
+# [PROJECT] Code of Conduct
+
+## The Short Version
+
+Be kind to others. Do not insult or put down others. Behave professionally. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate for [PROJECT].
+
+All communication should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate.
+
+[PROJECT] is dedicated to providing a harassment-free community for everyone, regardless of gender, sexual orientation, gender identity and expression, disability, physical appearance, body size, race, or religion. We do not tolerate harassment of community members in any form.
+
+Thank you for helping make this a welcoming, friendly community for all.
+
+**CODE OF CONDUCT REPORTING FORM:** [*https://numfocus.typeform.com/to/ynjGdT*](https://numfocus.typeform.com/to/ynjGdT)
```

Comment:
Projects should host their own reporting form, since they would be running their own CoC adjudication process.